[BFT Testing] Implements Corruptible Conduits (#2033)
onflow/flow-go: insecure/readme.md

# Insecure Package

> The insecure package encapsulates tools and technologies related to testing codebase against

Reviewer: ```suggestion
The insecure package encapsulates tools and technologies for testing codebase against
```

attack vectors of malicious nodes. The package and sub-packages namings are chosen in a way that reflect

> this purpose. This package should not be utilized for any production-grade development that affects

Reviewer: ```suggestion
this purpose. This package if fore **testing only** and should **not**  be utilized for any production-grade development that affects
```
